iwyu_output.cc

```diff
@@ -605,6 +636,28 @@ void IwyuFileInfo::ReportForwardDeclareUse(SourceLocation use_loc,
   LogSymbolUse("Marked fwd-decl use of decl", symbol_uses_.back());
 }
 
+void IwyuFileInfo::ReportUsingDeclUse(SourceLocation use_loc,
+                                      const UsingDecl* using_decl,
+                                      const NamedDecl* target_decl,
+                                      bool in_cxx_method_body,
+                                      const char* comment) {  
+  for(auto&& saved_decl : using_decl_lines_) {
+    if(saved_decl.matches(using_decl)) {
+      saved_decl.set_referenced();
+      break;
+    }
+  }
+
+  ReportFullSymbolUse(use_loc, using_decl, in_cxx_method_body, comment);
```

Comment:
It'd be nice if you could add comments to these two Report-calls describing what they do.

As I'm reading it, this marks the using-decl itself as used, say if it comes from an included header, is that it?